src/MSCorEE.Desktop/MSCorEE.cs

```diff
@@ -72,5 +79,37 @@ public static partial class MSCorEE
         /// </returns>
         [DllImport(nameof(MSCorEE), PreserveSig = true)]
         public static extern unsafe int StrongNameFreeBuffer(byte* pbMemory);
+
+        [DllImport(nameof(MSCorEE), CharSet = CharSet.Unicode, PreserveSig = true)]
+        public static extern int CLRCreateInstance(
+            [In, MarshalAs(UnmanagedType.LPStruct)] Guid rclsid,
```

Comment:
Didn't knew about it either, per http://blogs.msdn.com/b/adam_nathan/archive/2003/04/23/56635.aspx it seem pretty specific to this use case :

> UnmanagedType.LPStruct is only supported for one specific case: treating a System.Guid value type as an unmanaged GUID with an extra level of indirection.